### DISCUSSION TITLE
Add send options 'send without sound' (and 'schedule message')

### DISCUSSION THREAD

**Discussion by doej1367** [element-hq/element-meta#813] [Ideas]
#### What would you like to do?
+ Send messages silently
+ Send messages sceduled

#### Why would you like to do it?
If you know your friend is in class or asleap and you don't want annoy them or wake them up with message pings, but still want to get the message to them so they can read it later. It is also usefull for any information in general, that does not require an immediate responce or announcements and reminders sceduled in the future.

#### How would you like to achieve it?
Like Telegram implemented it using a right click on the send button. With an option somewhere in the message input bar. Here is a screenshot of the Telegram feature I'm referring to.
![grafik](https://user-images.githubusercontent.com/19228364/199984836-4417a0f8-e350-4001-92bd-a2396cc39c85.png)

**Comment by daniellekirkwood** [comment]
@amshakal 👀
